src/flexbox.rs

```diff
@@ -1800,4 +1803,52 @@ mod tests {
         assert_eq!(constants.container_size, Size::ZERO);
         assert_eq!(constants.inner_container_size, Size::ZERO);
     }
+
+    #[test]
+    fn hidden_layout_should_hide_recursively() {
+        let mut taffy = Taffy::new();
+
+        let grandchild_00 = taffy
+            .new_leaf(FlexboxLayout { display: Flex, size: Size::from_points(50.0, 50.0), ..Default::default() })
```

Comment:
Isn't display: Flex the default? And it should always stay that way.

Reply:
Yes `Display::Flex` is default. In the context of this test, I wanted to be explicit in showcasing that all children elements get hidden even though their Display is **not** set to `None`

Reply:
Ah I see. Okay, I'm cool with that.